Makefile

```diff
@@ -8,9 +13,34 @@ build-traffic-generator:
 	@echo About to build the Kiali Test Traffic Generator
 	make -C traffic-generator clean docker-build
 
-openshift-deploy:
-	@echo About to deploy a few sample projects to OpenShift
-	./hack/openshift-deploy.sh -n kiali-test-depth -c hack/test-configs/test-depth.yaml
-	./hack/openshift-deploy.sh -n kiali-test-breadth -c hack/test-configs/test-breadth.yaml
-	./hack/openshift-deploy.sh -n kiali-test-hourglass -c hack/test-configs/test-hourglass.yaml
-	./hack/openshift-deploy.sh -n kiali-test-circle-callback -c hack/test-configs/test-circle-callback.yaml
+openshift-deploy-kiali-test-depth:
+	@echo About to deploy Kiali Test Depth to OpenShift
+	ansible-playbook ./test-service/deploy/ansible/deploy_test_meshes.yml -e number_of_services=${NUM_SERVICES} -e number_of_versions=${NUM_VERSIONS} -e AVAILABLE_MESHES ?= '{"meshes": ["kiali-test-depth"]}' -v
+
+openshift-deploy-kiali-test-breath:
+	@echo About to deploy Kiali Test Breath to OpenShift
+	ansible-playbook ./test-service/deploy/ansible/deploy_test_meshes.yml -e number_of_services=${NUM_SERVICES} -e number_of_versions=${NUM_VERSIONS} -e AVAILABLE_MESHES ?= '{"meshes": ["kiali-test-breath"]}' -v
+
+openshift-deploy-kiali-test-circle:
+	@echo About to deploy Kiali Test Circle to OpenShift
+	ansible-playbook ./test-service/deploy/ansible/deploy_test_meshes.yml -e number_of_services=${NUM_SERVICES} -e number_of_versions=${NUM_VERSIONS} -e AVAILABLE_MESHES ?= '{"meshes": ["kiali-test-circle"]}' -v
+
+openshift-deploy-kiali-test-circle-callback:
+	@echo About to deploy Kiali Test Circle Callback to OpenShift
+	ansible-playbook ./test-service/deploy/ansible/deploy_test_meshes.yml -e number_of_services=${NUM_SERVICES} -e number_of_versions=${NUM_VERSIONS} -e AVAILABLE_MESHES ?= '{"meshes": ["kiali-test-circle-callback"]}' -v
+
+openshift-deploy-kiali-test-hourglass:
+	@echo About to deploy Kiali Test Hourglass to OpenShift
+	ansible-playbook ./test-service/deploy/ansible/deploy_test_meshes.yml -e number_of_services=${NUM_SERVICES} -e number_of_versions=${NUM_VERSIONS} -e AVAILABLE_MESHES ?= '{"meshes": ["kiali-test-hourglass"]}' -v
+
+openshift-deploy-kiali-test-depth-sink:
+	@echo About to deploy Kiali Test Depth Sink to OpenShift
+	ansible-playbook ./test-service/deploy/ansible/deploy_test_meshes.yml -e number_of_services=${NUM_SERVICES} -e number_of_versions=${NUM_VERSIONS} -e AVAILABLE_MESHES ?= '{"meshes": ["kiali-test-depth-sink"]}' -v
+
+openshift-deploy-kiali-test-breath-sink:
+	@echo About to deploy Kiali Test Breath Sink to OpenShift
+	ansible-playbook ./test-service/deploy/ansible/deploy_test_meshes.yml -e number_of_services=${NUM_SERVICES} -e number_of_versions=${NUM_VERSIONS} -e AVAILABLE_MESHES ?= '{"meshes": ["kiali-test-breath-sink"]}' -v
```

Comment:
i'll try it, I am surprised that this syntax works:

`-e AVAILABLE_MESHES ?= '{"meshes": ["kiali-test-breath-sink"]}'

Reply:
I didn't think that would work :) That syntax is incorrect - ansible-playbook's "-e" option doesn't like that `VER ?= value` -- I get this:

```
$ make openshift-deploy-kiali-test-breath
About to deploy Kiali Test Breath to OpenShift
ansible-playbook ./test-service/deploy/ansible/deploy_test_meshes.yml -e number_of_services=1 -e number_of_versions=1 -e AVAILABLE_MESHES ?= '{"meshes": ["kiali-test-breath"]}' -v
Using /etc/ansible/ansible.cfg as config file
ERROR! the playbook: ?= could not be found
```

In all those make targets (except for the "deploy all") take out the reference to AVAILABLE_MESHES. If I run "openshift-deploy-kiali-test-hourglass" I want it to  ONLY deploy hourglass - regardless of the value of AVAILABLE_MESHES (which, btw, is going to be set to all of them, as you define it up on line 1)

Reply:
In other words:

```
ansible-playbook ./test-service/deploy/ansible/deploy_test_meshes.yml -e number_of_services=${NUM_SERVICES} -e number_of_versions=${NUM_VERSIONS} -e '{"meshes": ["kiali-test-breath-sink"]}' -v
```

for the target that deploys only the breath-sink. Do the same for the others.